spec/rails_helper.rb

```diff
@@ -2,7 +2,10 @@
 
 ENV['RAILS_ENV'] ||= 'test'
 
-unless ENV['DISABLE_SIMPLECOV'] == 'true'
+# This needs to be defined before Rails is initialized
+RUN_SYSTEM_SPECS = ENV.fetch('RUN_SYSTEM_SPECS', false)
+
+if ENV['COVERAGE'] == 'true'
```

Comment:
Switching this around makes sense IMHO, but it also needs changes to the github workflow files.